### PR TITLE
feat(Constants): add discord faces

### PR DESCRIPTION
**Pull request by san4ern**
I've added discord faces from in-built slash commands

### CODE REVIEW

File: src/constants/common/DiscordFaces.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export enum DiscordFaces {
```
Comment (kislball) [inline src/constants/common/DiscordFaces.ts]: Add here some documentation since it might not be obvious for some people

Comment (kislball) [reply src/constants/common/DiscordFaces.ts]: I think you should also change name DiscordFaces to DiscordTextFaces

**Review by kislball — CHANGES_REQUESTED** [review]
There are some changes needed

**Review by kislball — APPROVED** [review]
looks nice, merging

**Review by kislball — CHANGES_REQUESTED** [review]
add an export to constants/common/index.ts

**Review by san4ern — COMMENTED** [review]
Made it

**Review by Mirdukkk — APPROVED** [review]

**Pull request merged**